collect_app/src/test/java/org/odk/collect/android/activities/SplashScreenActivityTest.java

```diff
@@ -52,41 +47,23 @@ public void setup() {
         RobolectricHelpers.mountExternalStorage();
         RobolectricHelpers.overrideAppDependencyModule(new AppDependencyModule() {
             @Override
-            public PermissionsProvider providesPermissionsProvider(PermissionsChecker permissionsChecker, StorageStateProvider storageStateProvider) {
-                return new AlwaysGrantStoragePermissionsPermissionsProvider(permissionsChecker, storageStateProvider);
-            }
-
-            @Override
-            public ApplicationInitializer providesApplicationInitializer(Application application, UserAgentProvider userAgentProvider, SettingsPreferenceMigrator preferenceMigrator, PropertyManager propertyManager, Analytics analytics, AppStateProvider appStateProvider, StorageStateProvider storageStateProvider) {
+            public ApplicationInitializer providesApplicationInitializer(Application application, UserAgentProvider userAgentProvider,
+                                                                         SettingsPreferenceMigrator preferenceMigrator, PropertyManager propertyManager,
+                                                                         Analytics analytics, StorageInitializer storageInitializer) {
                 return applicationInitializer;
             }
         });
     }
 
     @Test
-    public void whenStoragePermissionGranted_createsODKDirectories() {
+    public void whenAppStarts_createsODKDirectories() {
```

Comment:
Could we just move this to a test for `ApplicationIntitializer`? Alternatively we could just think of it as being implicitly tested by our Espresso test. Your call!

Reply:
I moved it to StorageInitializerTest.